tests/socialLogin.test.js

```diff
@@ -0,0 +1,96 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import express from 'express';
+import 'dotenv/config';
+import AuthPassportController from '../src/controllers/authentication.controller';
+import authMockData from './data/passportAuthMockData';
+import statusCode from '../src/utils/statusCodes';
+import customMessages from '../src/utils/customMessages';
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+const app = express();
+const router = express.Router();
+
+app.use(express.json());
+
+router.post('/facebook', AuthPassportController.facebookLogin);
+router.post('/google', AuthPassportController.googleLogin);
+
+app.use(
+  '/api/auth',
+  (req, res, next) => {
+    req.user = req.body;
+    next();
+  },
+  router
+);
+
+const facebookObject = authMockData[0];
+const googleObject = authMockData[1];
+
+describe('Passport Authentication controller', () => {
+  it('should authenticate a facebook user', (done) => {
+    chai
+      .request(app)
+      .post('/api/auth/facebook')

```

Comment:
are you using the same route for testing? you should create test routes for testing

Reply:
Am not using the same routes, this one is a Post but in the controller am using a Get. so different routes for tests and controller

Reply:
Please use descriptive names for those test routes, it will help everyone who reads your code to know that they are only there for testing.

Reply:
Done. I think it should be easy to distinguish the test routes from others now.Thanks